libs/core-ui/src/lib/Cohort/Cohort.ts

```diff
@@ -96,7 +96,14 @@ export class Cohort {
   }
 
   public getRow(index: number): { [key: string]: number } {
-    return { ...this.jointDataset.dataDict?.[index] };
+    const dataDict = this.jointDataset.dataDict?.[index];
+    const convertedDataDict: { [key: string]: number } = {};
+    if (dataDict) {
+      for (const key in dataDict) {
+        convertedDataDict[key] = Number(dataDict[key]);
```

Comment:
since we made a separate strDataDict I'm not sure if this conversion code is needed anymore - I think it can be removed now?